tests/spine_db_editor/test_shortcuts.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from unittest.mock import MagicMock, patch
+from PySide6.QtGui import QShortcut, QKeySequence
+
+
+class TestSpineDBEditorShortcuts(unittest.TestCase):
+    def setUp(self):
```

Comment:
Test is missing `QApplication` setup. See how other Toolbox tests are set up. You may want to inherit the test from `DBEditorTestBase`, not `unittest.TestCase`.